Makefile

```diff
@@ -1,19 +1,27 @@
+CONTAINER_NAME=packit
+CONTAINER_EXEC=podman exec $(CONTAINER_NAME)
+
 prepare-check:
 	ansible-playbook -b -K -i inventory-local -c local ./recipe-tests.yaml
 
-check:
-	tox
+container:
+	rsync -a $(CURDIR)/ /tmp/packit
+	podman container inspect packit || podman run --name $(CONTAINER_NAME) -ti -d -v /tmp/packit:/src:Z -w /src fedora:30
+	sleep 2
+	$(CONTAINER_EXEC) bash -c "dnf -y install make"
+
+container_kill:
+	podman stop $(CONTAINER_NAME) || true
+	podman rm -f $(CONTAINER_NAME)
+
+test_build:
+	. tests/testlib.sh; test_build
 
-# build-tests: recipe-tests.yaml
-# 	ansible-bender build -- ./recipe-tests.yaml $(SOURCE_GIT_IMAGE) $(PACKIT_TESTS_IMAGE)
+test_install:
+	. tests/testlib.sh; test_install
 
-# shell:
-# 	podman run --rm -ti -v $(CURDIR):/src:Z -w /src $(PACKIT_TESTS_IMAGE) bash
+check: test_install
```

Comment:
no offense but I ain't gonna run this locally when writing new code: rebuilding packit RPM is really an overkill; especially when packit service does this for you

Reply:
Is  there any possibility how to install dependencies? build requires could be done via ``dnf -y builddep `` 
but not sure if there is possibility to install runtime deps based on specfile.

Otherwise this also test installation and removing built package, so that this extends what packit-service does.

Reply:
`ansible-playbook -i localhost, files/install-rpm-packages.yaml`